bzt/modules/jmeter.py

```diff
@@ -1722,20 +1776,21 @@ def __add_managers(self):
         if self.scenario.get("store-cookie", True):
             self.append(self.TEST_PLAN_SEL, self._get_cookie_mgr())
             self.append(self.TEST_PLAN_SEL, etree.Element("hashTree"))
+        self.append(self.TEST_PLAN_SEL, self._get_dns_cache_mgr())
```

Comment:
if scenario.get(use-dns-cache-mgr):
   # add manager
   # append ttl=0 to system.props